### PR TITLE
added Zvanu Parvaldnieks

### PR DESCRIPTION
**Pull request by einarshamrihs**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @einarshamrihs, thank you so much for submitting your pull request. This is a new verified connector. Like all verified publisher connectors, we ask that you register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration). 
Please let me know if you have any questions.
Thank you very much again for working with us.

**Comment by Amjed-Ayoub** [comment]
Hi @einarshamrihs can you please register your connector, so we can review the pull request and move forward with the certification process.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hello @einarshamrihs, thank you so much for submitting your pull request. This is a new verified connector. Like all verified publisher connectors, we ask that you register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration).
Please let me know if you have any questions.
Thank you very much again for working with us.

File: certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3331 @@
...
+                      "description": "employee_id"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "description": "Retreive a single voip endpoint object by providing an ID.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json:1638]: @einarshamrihs please make sure to capitalize abbreviations if you use them to make everything clear. E.g. `VoIP` You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3331 @@
...
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "endpoint": {
+                  "type": "string",
+                  "description": "endpoint",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json:2515]: @einarshamrihs empty properties are not allowed.

File: certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3331 @@
...
```
Comment (v-EgorKozhadei) [inline certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json:2893]: @einarshamrihs empty properties are not allowed.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @einarshamrihs! Please check comments after the review, thank you!

File: certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,3491 @@
...
+            "name": "date_to",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "date-time",
+            "description": "End date in the filtered date range.",
+            "x-ms-summary": "Date to"
+          }
+        ],
+        "x-ms-trigger-hint": "Triggers when new call recording is created",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json:631]: @einarshamrihs please make sure everything is grammatically correct and free of typographical errors, you can follow [these guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @einarshamrihs! Please check a few more comments, we'll be good to go after they're fixed, thank you!

File: certified-connectors/Zvanu parvaldnieks/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+          },
+          "x-ms-apimTemplateParameter.triggerDataPath": "@triggerBody().data",
+          "x-ms-apimTemplate-operationName": [
+            "GetCallHistory"
+          ]
+        }
+      },
+      {
+        "templateId": "pollingtrigger",
+        "title": "PollingTriggerPolicy_714c4ee7-2d02-d48b-6334-ec4290ccd17c",
```
[the remark this answers is not in the document]
Comment (einarshamrihs) [reply certified-connectors/Zvanu parvaldnieks/apiProperties.json]: Those were generated automatically, when I downloaded project with paconn command. I replaced them now.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @einarshamrihs, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @einarshamrihs,

I hope you are doing well.

This is to inform you that we will reschedule our deployment for production until Tuesday June 28th & Wednesday June 29th. 
Your connector is in our registration backlog with the status "In Partner Verification/Testing". 
We have high volume of connectors in our pipeline. Therefore, we need to clear our backlog. We tested your connector, and everything looks good on our side. We would like to proceed forward with Production Deployment for your connector 😊.

Please let us know if you want us to stop deploying your connector or if you still need time for testing. 

Feel free to reach out with any questions you may have.

I look forward to hearing from you soon.

Thank you so much for understanding.

Best regards,
Amjed Ayoub

**Pull request merged**